test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -562,5 +569,21 @@ private IEnumerable<FunctionMetadata> GetTestFunctionsList(string runtime)
                 }
             };
         }
+
+        private void VerifyStartIntervals(bool ignoreFirstStart = false)
+        {
+            // Verify that interval beetwen starts >10 secs and <15 secs
+            var startTimestamps = _testLogger.GetLogMessages().Where(x => x.FormattedMessage
+                .Contains("RegisterFunctions called")).Select(x => x.Timestamp).ToList();
+            if (ignoreFirstStart)
+            {
+                startTimestamps.RemoveAt(0);
+            }
+            for (int i = 1; i < startTimestamps.Count(); i++)
+            {
+                var diff = startTimestamps[i] - startTimestamps[i - 1];
+                Assert.True(diff > TimeSpan.FromSeconds(10) && diff < TimeSpan.FromSeconds(15));
```

Comment:
I would consider making the hardcoded values (10 seconds and 15 seconds) parameters, so that readers of the process start unit tests have an idea of what VerifyStartIntervals is doing without having to read through the implementation.

Reply:
Fixed.